src/gt4py/testing/__init__.py

```diff
@@ -14,6 +14,7 @@
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
 
+__all__ = ["field", "global_name", "none", "parameter", "StencilTestSuite"]
```

Comment:
This is not really necessary since the exported names are the ones that were explicitly collected from submodules and all of them are public (not starting with an underscore).

Reply:
Just a comment, though. Feel free to leave it as it is.

Reply:
If you don't put it, flake8 complains about F401 (imported but unused)

Reply:
Ah, ok, makes sense.